istresearch/rapidpro
# P4 1523 - Engage Permissions: Ensure that editors can view the channels list page. Editors can only view channels, and cannot add, edit, or remove them.

# For the Reviewer

When this is complete, you should approve the PR via github.

# For the Reviewee

<Reminder PR Title should be JIRA_NUMBER and Useful Description>

## Summary
Description of changes.
Editors can view the channels list page. Editors can only view channels, and cannot add, edit, or remove them.

#### Release Note
Editors can view the channels list page. Editors can only view channels, and cannot add, edit, or remove them.

#### Breaking Changes
N/A

## Quality Assurance

You have gathered the following items:

Given all of the items above, you have updated your Application ORT at the following locations:
- **Features and Alerting**: <link to app ORT>Required.
- **P4 Alerting**: <link to p4-alerting PR>Required.

## Testing and Verification

- Standup Engage
- Create one admin user, create a separate Editor user.
- Navigate to: /channels/channel/manage/
- The Editor should be able to see the list, but should not see any menu icons to add additional channels.
- The Admin should see the list, along with the menu icons to add additional channels.
- The Admin should be able to navigate to /channels/channel/claim/
- The Editor should. not be able to navigate directly to /channels/channel/claim/
